samples/bluetooth/peripheral_ht/src/hts.c

```diff
@@ -25,7 +25,11 @@
 #include <bluetooth/uuid.h>
 #include <bluetooth/gatt.h>
 
+#ifdef CONFIG_SOC_FAMILY_NRF
```

Comment:
The way it is set up now this should be `#ifdef CONFIG_TEMP_NRF5`.
The application can still disable this option, and that would lead to compilation error.

```
west build samples/bluetooth/peripheral_ht -- -DCONFIG_TEMP_NRF5=n
hts.c:29: undefined reference to `__device_dts_ord_46'`
```

Reply:
> The application can still disable this option, and that would lead to compilation error.

Good catch, thank you. Now fixed along with a couple of other minor improvements.